tests/test_bounding_box.py

```diff
@@ -384,19 +386,19 @@ def test_bbox_get_config_from_config():
     assert bb2_config == bb1_config
 
 
-def test_bbox_ndim(ndim):
+def test_bbox_ndim(ndim: Tuple[int, int]) -> None:
```

Comment:
This should just be an int and is also not consistent with the return type documented for the `ndim` fixture above.
```suggestion
def test_bbox_ndim(ndim: int) -> None:
```